static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -286,8 +286,9 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
       case 'good':
         return {
           color: 'green300',
-          description: tct('[value]ms (Good)', {
+          description: tct('[value][ms] (Good)', {
             value: frame.data.value.toFixed(2),
+            ms: frame.description === 'cumulative-layout-shift' ? '' : 'ms',
```

Comment:
small nit: could move this into a constant to avoid having to type this ternary out 3 times